control-plane/go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	k8s.io/apimachinery v0.22.2
 	k8s.io/client-go v0.22.2
 	k8s.io/klog/v2 v2.9.0
+	k8s.io/utils v0.0.0-20220812165043-ad590609e2e5
```

Comment:
This seems related to changes not made in the PR but caused automatically by a `go mod tidy`.

Reply:
Hrm, this was a change I made yesterday for the `pointer` changes.. I wonder how that worked..

Reply:
yeah..not quiet sure myself